docs/pages/repo/docs/support.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Support Policy
+description: Our Support Policy for Turborepo and its ecosystem
+---
+
+## Support Policy
+
+`turbo` is generally supported on Debian-based Linux distributions, macOS, and
+Windows on both x86_64 and ARM 64 architectures. Specifically, we build and ship
+the following binaries via `npm`:
+
+- turbo-darwin-64 (macOS with Intel chips)
+- turbo-darwin-arm64 (macOS with Apple Silicon)
+- turbo-linux-64
+- turbo-linux-arm64
+- turbo-windows-64
+- turbo-windows-arm64
+
+darwin builds are intended for macOS on Intel and Apple Silicon. The linux
+builds are tested on Ubuntu, but we expect them to work on most
+[Debian-based][3] distros. Windows builds are tested on the
+[`windows-latest` runner][2] in GitHub Actions.
+
+## Node.js Compatibility
+
+Most core `turbo` functionality (especially `turbo run`) does not depend on the
+active Node.js version on your system, but some features of Turborepo and its ecosystem
+such as `create-turbo`, `turbo-ignore`, and `eslint-plugin-turbo` do. For these features,
+we intend to support the [Active and Maintenance LTS versions of Node.js][1].
+
+## Package Managers
+
+Core `turbo` functionality depends on the package managers in the JS ecosystem
+and their implementations of workspace configuration (in monorepos) and
+lockfiles formats. We intend to support:
+
+- `npm` (v6+)
+- `yarn` (v1+)
+- `pnpm` (v6+)
+- `bun` (v1+)
```

Comment:
We don't support `prune` for `bun`. Maybe it needs an asterisk?

Reply:
good call,updated